public/i18n/qqq.json

```diff
@@ -1,3 +1,16 @@
 {
-  "query-builder-run-query": "Text on the button that runs the query"
+  "query-builder-heading": "Title of the page",
+  "query-builder-description": "A short description on what query builder is",
+  "query-builder-find-all-items": "header for part of the form finding all items",
+  "query-builder-run-query": "Text on the button that runs the query",
+  "query-builder-input-value": "Text of value of the property in the form",
```

Comment:
This name should be `query-builder-input-value-label` or maybe `query-builder-value-input-label`. Also the explanation could be a bit clearer. What about "Label of the value input" or "Label of the property value input"?

Reply:
You're right. I changed it. I changed it to `query-builder-input-value-label` as more I wanted it to have tree like structure. "label of value of input part of query builder"

Reply:
Makes sense. Thanks!